ngrinder-controller/src/main/resources/script_template/groovy_maven/pom.xml

```diff
@@ -6,7 +6,7 @@
 	<version>0.0.1</version>
 
 	<properties>
-		<ngrinder.version>3.4</ngrinder.version>
+		<ngrinder.version>3.4.2-SNAPSHOT</ngrinder.version>

```

Comment:
This should be updated and the core should be released on maven repo when releasing 3.4.2